features/verblijfplaatshistorie.feature

```diff
@@ -1,182 +1,626 @@
-
-  Scenario: Historische verblijfplaats is in onderzoek
-    Als de ingeschreven persoon met burgerservicenummer 999990378 wordt geraadpleegd met peildatum 2010-01-01
-    Dan bevat verblijfplaats met straatnaam Rapheëlstraat inOnderzoek
-
-  Scenario: Historische verblijfplaats is in onderzoek geweest
-    Als de ingeschreven persoon met burgerservicenummer 999990263 wordt geraadpleegd met peildatum 2010-01-01
-    Dan bevat verblijfplaats met straatnaam Rapheëlstraat geen inOnderzoek
+    Als verblijfplaatshistorie wordt geraadpleegd met de volgende parameters
+    | naam                | waarde             |
+    | type                | RaadpleegMetPeriode|
+    | burgerservicenummer | 999994669          |
+    | fields              | datumVan, datumTot, datumIngangGeldigheid, adresregel1, land.code     |
+    Dan heeft de response een object met de volgende gegevens
+    | naam     | waarde                                                      |
+    | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1 |
+    | title    | Minimale combinatie van parameters moet worden opgegeven.   |
+    | status   | 400                                                         |
+    | detail   | De foutieve parameter(s) zijn: datumVan, datumTotEnMet      |
+    | code     | paramsCombination                                           |
+    | instance | /haalcentraal/api/brp/verblijfplaatshistorie                |
+    En heeft het object de volgende 'invalidParams' gegevens
+    | code     | name          | reason                  |
+    | required | datumVan      | Parameter is verplicht. |
+    | required | datumTotEnMet | Parameter is verplicht. |
+
+  @fout-case
+  Scenario: Verblijfplaats raadplegen met lege peildatum
+    Als verblijfplaatshistorie wordt geraadpleegd met de volgende parameters
+    | naam                | waarde               |
+    | type                | RaadpleegMetPeildatum|
+    | burgerservicenummer | 999994669            |
+    | fields              | datumVan, datumTot, datumIngangGeldigheid, adresregel1, land.code |
+    | peildatum           | -                    |
+    Dan heeft de response een object met de volgende gegevens
+    | naam     | waarde                                                      |
+    | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1 |
+    | title    | Minimale combinatie van parameters moet worden opgegeven.   |
+    | status   | 400                                                         |
+    | detail   | De foutieve parameter(s) zijn: peildatum                    |
+    | code     | paramsCombination                                           |
+    | instance | /haalcentraal/api/brp/verblijfplaatshistorie                |
+    En heeft het object de volgende 'invalidParams' gegevens
+    | code     | name          | reason                  |
+    | required | peildatum     | Parameter is verplicht. |
```

Comment:
peildatum is wel geleverd, maar heeft geen geldige waarde (is geen datum). ik verwacht dus een andere foutmelding.